modules/programs/firefox.nix

```diff
@@ -301,410 +301,418 @@ in {
-                      "briefcase"
-                      "cart"
-                      "circle"
-                      "dollar"
-                      "fence"
-                      "fingerprint"
-                      "gift"
-                      "vacation"
-                      "food"
-                      "fruit"
-                      "pet"
-                      "tree"
-                      "chill"
-                    ];
-                    default = "fruit";
-                    description = "Container icon.";
-                  };
-                };
-              }));
-              default = { };
-              example = {
-                "shopping" = {
-                  id = 1;
-                  color = "blue";
-                  icon = "cart";
-                };
-                "dangerous" = {
-                  id = 2;
-                  color = "red";
-                  icon = "fruit";
-                };
+                  To automatically enable extensions add
+                  `"extensions.autoDisableScopes" = 0;`
+                  to
+                  [{option}`programs.firefox.profiles.<profile>.settings`](#opt-programs.firefox.profiles._name_.settings)
+                '';
               };
-              description = ''
-                Attribute set of container configurations. See
-                [Multi-Account
-                Containers](https://support.mozilla.org/en-US/kb/containers)
-                for more information.
-              '';
-            };
 
-            extensions = mkOption {
-              type = types.listOf types.package;
-              default = [ ];
-              example = literalExpression ''
-                with pkgs.nur.repos.rycee.firefox-addons; [
-                  privacy-badger
-                ]
-              '';
-              description = ''
-                List of Firefox add-on packages to install for this profile.
-                Some pre-packaged add-ons are accessible from the
-                [Nix User Repository](https://github.com/nix-community/NUR).
-                Once you have NUR installed run
-
-                ```console
-                $ nix-env -f '<nixpkgs>' -qaP -A nur.repos.rycee.firefox-addons
-                ```
-
-                to list the available Firefox add-ons.
-
-                Note that it is necessary to manually enable these extensions
-                inside Firefox after the first installation.
-
-                To automatically enable extensions add
-                `"extensions.autoDisableScopes" = 0;`
-                to
-                [{option}`programs.firefox.profiles.<profile>.settings`](#opt-programs.firefox.profiles._name_.settings)
-              '';
             };
 
-          };
-        }));
+            config = {
+              settings."toolkit.legacyUserProfileCustomizations.stylesheets" =
+                mkIf (profile.userChrome != "") true;
+            };
+          }));
```

Comment:
Thanksl! :blush: 

I also just verified that `userContent` also needs this option, so I think it should be included here.

Reply:
It's the same options?? o.0 Cause it says "stylesheets" there.

Reply:
Ah, nvm, I was thinking that userContent refers to `userContent.js`.